.gitignore

```diff
@@ -46,3 +46,5 @@ hs_err_pid*
 build/
 i18n/
 packages/**/yarn.lock
+packages/containers/sandbox/dist
+packages/containers/sandbox/package-lock.json
```

Comment:
Those should be in a `.gitignore` file in sandbox/
It will be hard to maintain otherwise (ex: we remove sandbox, this has a lot of chance to stay)